uk_bin_collection/uk_bin_collection/councils/NorthTynesideCouncil.py

```
@@ -1,138 +1,77 @@
-import math
-from datetime import *
-
-import requests
+import logging
+from datetime import datetime
 from bs4 import BeautifulSoup
 
 from uk_bin_collection.uk_bin_collection.common import *
```

Ruff (0.14.0)

`from uk_bin_collection.uk_bin_collection.common import *` used; unable to detect undefined names

(F403)